fdbserver/Knobs.cpp

```diff
@@ -84,7 +84,7 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 	init( DISK_QUEUE_MAX_TRUNCATE_BYTES,                       2<<30 ); if ( randomize && BUGGIFY ) DISK_QUEUE_MAX_TRUNCATE_BYTES = 0;
 	init( TLOG_DEGRADED_DURATION,                                5.0 );
 	init( MAX_CACHE_VERSIONS,                                   10e6 );
-	init( TLOG_IGNORE_POP_AUTO_ENABLE_DELAY,                   300.0 );
+	init( TLOG_IGNORE_POP_AUTO_ENABLE_DELAY,                   300.0 ); if ( randomize && BUGGIFY ) TLOG_IGNORE_POP_AUTO_ENABLE_DELAY = 60.0;
```

Comment:
I don't think `randomize` matters here, we could run into this same issue with `randomize == false`. We either want to adjust this knob when buggify is enabled or whenever we're running in simulation.